### ISSUE TITLE
A5

### ISSUE THREAD

**Issue by tpreusse** [orbiting/pdf#107]
The center element should have a tiny margin:
![a5](https://user-images.githubusercontent.com/410211/39725999-e5884ee6-524d-11e8-9d1d-94b7a6758b03.png)

Otherwise just a bunch of media queries?

**Comment by diegomura** [comment]
Sorry 😅 . What do you mean by tiny margin? a narrow margin?
We can definitely use media queries for this! Would love to see them in action. If we don't use them, there's no way of adjusting A5 styles without changing A4 also.

**Comment by tpreusse** [comment]
@diegomura yeah meant narrow margins—narrow then with A4. I updated the issue the be a bit clearer.

Do you have an media query example?

**Comment by diegomura** [comment]
Cool. Totally doable with media queries. 
You can see an example here: https://github.com/diegomura/react-pdf/blob/master/examples/resume/index.js

Currently you cannot query by page size (only min/max width/height and orientation), but you can workaround this by using W5 dimensions right away
